src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -39,7 +37,7 @@ internal class KafkaConsumerActor<K, V> : ActorBase, ILogReceive, IWithTimers
         /// <summary>
         /// Stores delegates for external handling of partition events
         /// </summary>
-        private readonly IPartitionEventHandler _partitionEventHandler;
+        private IPartitionEventHandler _partitionEventHandler;
```

Comment:
The `_partitionEventHandler` will now be assigned via message-handling rather than through the CTOR